parts/22-query.sh

```diff
@@ -1161,6 +1161,12 @@ query_tool-use-by-group() { ##? <year_month> <group>: Lists count of tools used
 
 EOFhelp
 	username=$(gdpr_safe galaxy_user.username username "Anonymous User")
+
+	group_filter=""
+	if (( $# > 0 )); then
+		group_filter="AND galaxy_group.name = '$arg_group'"
+	fi
+
```

Comment:
```suggestion

	group_filter=""
	if [[ ! -n "$arg_group" ]] then
		group_filter="AND galaxy_group.name = '$arg_group'"
	fi

```

Relying on argument counts is not ideal here, testing whether the variable is defined is a lot more useful.

Could you change the signature of this function to be

```
##? <years_month> [--group=<name>]
```

Which will

1. Indicate that the group is optional, rather than required (`<>`)
2. And ensure that the we can add more arguments later, without interfering with this one.

Reply:
Agreed - however, I decided to remove this filter because it was unnecessary for Tool Performance Tracking needs at this time.